homeassistant/components/update/__init__.py

```diff
@@ -0,0 +1,253 @@
+"""Support for Update."""
+from __future__ import annotations
+
+import asyncio
+import dataclasses
+import logging
+from typing import Any, Protocol
+
+import async_timeout
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import integration_platform, storage
+from homeassistant.helpers.typing import ConfigType
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "update"
+INFO_CALLBACK_TIMEOUT = 5
+STORAGE_VERSION = 1
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Update integration."""
+    hass.data[DOMAIN] = UpdateManager(hass=hass)
+
+    websocket_api.async_register_command(hass, handle_info)
+    websocket_api.async_register_command(hass, handle_update)
+    websocket_api.async_register_command(hass, handle_skip)
+
+    await integration_platform.async_process_integration_platforms(
```

Comment:
We can do this when `UpdateManager` loads. That way saves some more work during startup.

Reply:
It does not load.

Reply:
I mean when you load the skipped data from the store.